main.py

```diff
@@ -355,11 +347,12 @@ def CRYTERIA():
         else:
             pass
 
+
```

Comment:
Pierdoła, ale raczej nie dodawaj zbędnych linii w kodzie. Nie wiem z jakiego edytora korzystasz ale możesz zainteresować się jakimś linterem np. https://code.visualstudio.com/docs/python/linting, który wykrywa takie pierdoły a jak odpowiednio sobie IDE skonfigurujesz to może i nawet automatycznie ci takie pierdoły wyłapie i poprawi

Reply:
Z Pycharma, ale dodałem sobie już to rozszerzenie do VsCode